api/app/tests/apps/api/v1/test_atomicity_of_updates.py

```diff
@@ -164,6 +165,7 @@ def test_partial_fail_no_django_signals_fired(self, mocked):
         update_category_assignment.connect(fake_receiver)
         update_priority.connect(fake_receiver)
         create_note.connect(fake_receiver)
+        update_type.connect(fake_receiver)
```

Comment:
Add a Signal type change to the test data (`self.payload`) to demonstrate that the presence of type change payload does not lead to a call into the `fake_receiver`. Current addition to the test is moot because the type codepath will not be called (no `type` payload).

Reply:
Added